2022/39xxx/CVE-2022-39390.json

```diff
@@ -1,9 +1,8 @@
 {
```

Comment:
{} is not allowed for {'vendor': {'vendor_data': [{'product': {'product_data': [{'version': {'version_data': [{}]}}]}}]}}